modules/utxo-lib/src/bitgo/Unspent.ts

```diff
@@ -0,0 +1,106 @@
+import { TxOutput } from '..';
+import { Network } from '../networkTypes';
+import { toOutputScript } from '../address';
+import { UtxoTransactionBuilder } from './UtxoTransactionBuilder';
+
+/**
+ * Public unspent data in BitGo-specific representation.
+ */
+export interface Unspent {
+  /**
+   * Format: ${txid}:${vout}
+   */
+  id: string;
+  /**
+   * In network-specific representation
+   */
+  address: string;
+  /**
+   * In satoshi
+   */
+  value: number;
+}
+
+/**
+ * @return TxOutput from Unspent
+ */
+export function toOutput(u: Unspent, network: Network): TxOutput {
+  return {
+    script: toOutputScript(u.address, network),
+    value: u.value,
+  };
+}
+
+/**
+ * @param outputId
+ * @return TxOutPoint
+ */
+export function parseOutputId(outputId: string): TxOutPoint {
+  const parts = outputId.split(':');
+  if (parts.length !== 2) {
+    throw new Error(`invalid outputId, must have format txid:vout`);
+  }
+  const [txid, voutStr] = parts;
+  const vout = Number(voutStr);
+  if (txid.length !== 64) {
```

Comment:
Is it worthwhile to check here that the txid is valid hex?

Reply:
nah I don't think it is - this is just to catch the worst offenders

it gets checked further down the line in `addInput`